hinges.scad

```diff
@@ -508,8 +508,8 @@ module snap_lock(thick, snaplen=5, snapdiam=5, layerheight=0.2, foldangle=90, hi
     attachable(anchor,spin,orient, size=size) {
         back(snap_x) {
             cube([snaplen, snapdiam, snapdiam/2+thick], anchor=BOT) {
-                attach(TOP) xcyl(l=snaplen, d=snapdiam, $fn=16);
-                attach(TOP) xcopies(snaplen-snapdiam/4/3) xscale(0.333) sphere(d=snapdiam*0.8, $fn=12);
+                attach(TOP) xcyl(l=snaplen, d=snapdiam);
```

Comment:
My only caveat here is that we may want to enforce a *minimum* `$fn`.  Something like `$fn=max(16,segs(snapdiam/2))`.

Reply:
And possibly we want $fn to be a multiple of 4.

Reply:
OK, will do.  Thanks!

Reply:
Sorry for the delay.  I looked at the changes on my phone, but didn't want to write a comment on the phone's keyboard, and then I forgot it until now. 😞 

In any case, I just wanted to note the existence of the `quantup()`, `quantdn()`, and `quant()` functions for quantizing numbers a bit cleaner.